index.css

```diff
@@ -0,0 +1,174 @@
+body {
+  background-color: hsl(204, 43%, 93%);
+  height: 100vh;
+  font-family: Karla;
+  margin: 0;
+}
+
+.main-container {
+  display: flex;
+  justify-content: center;
+  min-height: 100vh;
+}
+
+.main-div-align-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.main-div {
+  max-width: 636px;
+  border-radius: 16px;
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  box-shadow: 0px 0px 20px rgba(0, 0, 0, 0.2);
+}
+
+.intro-section {
+  justify-content: center;
+  background-color: white;
+  padding: 0 40px;
+  height: 216px;
+}
+.intro-section h1 {
+  margin-top: 38px;
+  font-size: 24px;
+  color: #2ab2af;
+}
+.intro-section h2 {
+  color: #c0df34;
+  font-size: 18px;
+  margin: 24px 0px 0px;
+}
+.intro-section p {
+  color: #96a6bd;
+  font-size: 15px;
+  margin: 10px 0px 0px;
+  line-height: 26px;
+}
+
+.sub-part {
+  display: flex;
+  flex-direction: row;
+}
+
+.description-section {
+  display: flex;
+  flex-direction: column;
+  background-color: #2ab2af;
+  flex: 1;
+  padding: 40px;
+}
+.description-section h1 {
+  font-size: 18px;
+  color: #e5eff5;
+  margin: 0;
+}
+.description-section div {
+  display: flex;
+  flex-direction: row;
+  margin: 16px 0px 0px;
+  gap: 20px;
+}
+.description-section div h2 {
+  font-size: 30px;
+  color: #e5eff5;
+  margin: 0;
+}
+.description-section div p {
+  font-size: 16px;
+  margin: 8px 0px 0px;
+  color: #e5eff5;
+}
+.description-section p {
+  margin: 8px 0px 0px;
+  font-size: 16px;
+  color: #e5eff5;
+}
+.description-section button {
+  width: 100%;
+  padding: 17px;
+  margin: 30px 0px 0px;
+  font-size: 14px;
+  font-family: Arial;
+  color: white;
+  background-color: #c0df34;
+  border: none;
+  border-radius: 8px;
+  cursor: pointer;
+  font-weight: 600;
+  box-shadow: 0px 0px 10px rgba(0, 0, 0, 0.2);
+}
+.description-section button:hover {
+  background-color: #2fc6c4;
+}
+
+.about-section {
+  flex: 1;
+  background-color: #2fc6c4;
+  padding: 40px;
+  display: flex;
+  flex-direction: column;
+}
+.about-section h2 {
+  font-size: 18px;
+  color: #e5eff5;
+  margin: 0;
+}
+.about-section ul {
+  list-style: none;
+  margin: 17px 0px 0px;
+  padding: 0;
+  color: #e5eff5;
+  font-size: 14px;
+  line-height: 20px;
+}
+.about-section li {
+  position: relative;
+}
+.about-section li:before {
+  position: absolute;
+  content: "";
+  width: 15px;
+  height: 15px;
+  background-image: url(./assets/javascript.svg);
```

Comment:
for knowledge, if we add a data image then its very fast to load, so that image is preffered

Reply:
can you give an example for data image. you means to say that an svg image?